doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,661 @@
+    Data is now sent on all BISes when generated by the application (no SD card inserted).
+
+Bluetooth Fast Pair samples
+---------------------------
+
+* :ref:`fast_pair_locator_tag` sample:
+
+  * Updated the MCUboot bootloader configuration for the :ref:`zephyr:nrf54l15dk_nrf54l15` board target to enable the :kconfig:option:`CONFIG_FPROTECT` Kconfig option that is used to protect the bootloader partition against memory corruption.
+
+* :ref:`fast_pair_input_device` sample:
+
+  * Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target.
+
+Cellular samples
+----------------
+
+* Updated the :kconfig:option:`CONFIG_NRF_CLOUD_CHECK_CREDENTIALS` Kconfig option to be optional and enabled by default for the following samples:
+
+  * :ref:`nrf_cloud_rest_cell_location`
+  * :ref:`nrf_cloud_rest_device_message`
+  * :ref:`nrf_cloud_rest_fota`
+
+* :ref:`location_sample` sample:
+
+  * Updated:
+
+    * The Thingy:91 X build to support Wi-Fi by default without overlays.
+    * The Wi-Fi configurations to reduce the RAM usage by about 25 kB.
+
+* :ref:`modem_shell_application` sample:
+
+  * Updated the Wi-Fi configurations to reduce the RAM usage by about 25 kB.
+
+* :ref:`nrf_cloud_multi_service` sample:
+
+  * Updated the Wi-Fi configurations to reduce the RAM usage by about 12 kB for an nRF91 Series DK and 25 kB for the Thingy:91 X.
+
+DECT NR+ samples
+----------------
+
+* :ref:`dect_shell_application` sample:
+
+  * Added:
+
+    * The ``dect mac`` command.
+      A brief MAC-level code sample on top of DECT PHY interface with new commands to create a periodic cluster beacon, scan for it, associate or disassociate a PT/client, and send data to a FT/beacon random access window.
+      This is not a full MAC implementation and not fully compliant with DECT NR+ MAC specification (`ETSI TS 103 636-4`_).
+    * The ``startup_cmd`` command.
+      This command is used to store shell commands to be run sequentially after bootup.
+    * Band 4 support for nRF9151 with modem firmware v1.0.2.
+
+  * Updated:
+
+    * The ``dect rssi_scan`` command with busy/possible/free subslot count-based RSSI scan.
+    * The ``dect rx`` command to provide the possibility to iterate all channels and to enable RX filter.
+
+Edge Impulse samples
+--------------------
+
+* Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target in the following samples:
+
+  * :ref:`ei_data_forwarder_sample`
+  * :ref:`ei_wrapper_sample`
+
+Matter samples
+--------------
+
+* Updated:
+
+  * All Matter samples that support low-power mode to enable the :ref:`lib_ram_pwrdn` feature.
+    It is enabled by default for the release configuration of the following samples:
+
+    * :ref:`matter_lock_sample`
+    * :ref:`matter_light_switch_sample`
+    * :ref:`matter_smoke_co_alarm_sample`
+    * :ref:`matter_window_covering_sample`
+
+  * All Matter samples to enable the ZMS file subsystem in all devices that contain MRAM, such as the nRF54H Series devices.
```

Comment:
Hi @peknis, please duplicate (if it involves other devices too) this entry back to the changelog, it is relevant for 2.9-nRF54H20.